lib/internal/errors.js

```diff
@@ -737,6 +737,11 @@ E('ERR_NO_ICU', '%s is not supported on Node.js compiled without ICU');
 E('ERR_NO_LONGER_SUPPORTED', '%s is no longer supported');
 E('ERR_OUT_OF_RANGE', outOfRange);
 E('ERR_PARSE_HISTORY_DATA', 'Could not parse history data in %s');
+E('ERR_PATH_IS_DIRECTORY', (path) => {
+  if (require('internal/util/types').isUint8Array(path))
+    path = '';
+  return `Path ${path} cannot end with / in operations invalid on directories`;
```

Comment:
I try to understand the message but after reading it multiple times, I still can not fully understand it. Can you please rephrase this?

Reply:
It felt that way at first but I left it because am not a native English speaker and it was suggested by @joyeecheung.

Reply:
One way to make this more debuggable and consistent with other error messages is:

```
The "${name}" argument must not end with '/'. Received ${util.inspect(value)}.
```

That requires us to pass in the name of the argument into the formatter, like `ERR_INVALID_ARG_TYPE`, `ERR_OUT_OF_RANGE` and the like.